### ISSUE TITLE
[Suggestion] Hide (custom) fields

### ISSUE THREAD

**Issue by Snify89** [eikek/docspell#1658]
I love that you can assign custom fields to an item.
However it would be nice to hide certain fields (and not all of them - in the UI settings).

I use custom fields to store calendar IDs from Google.
The custom field is not really relevant for the user, but for the item itself.

It would be nice to hide custom fields in the item/result view.

Thanks for considering.

**Comment by eikek** [comment]
How would you assign values to these fileds then? Only via api?

**Comment by Snify89** [comment]
Yes only via API.  Not sure about all the client API stuff, but it would be a new setting for the collective or user, which saves, if the custom field is visible or not. This concerns only the restserver UI. Custom UIs that use the regular query item API, could just hide the custom fields if they'd want to, lol

**Comment by TheOneValen** [comment]
I have similar usecases, where I would like to hide the field, but use it in the API. Currently I have to hide all custom field, otherwise the overview would not be pretty to look at ...

**Comment by eikek** [comment]
I see, I think another flag for a custom field could be done in the settings which require to be reworked a bit to show every custom field and allow to toggle it.
